packages/react-core/src/components/Checkbox/Checkbox.tsx

```diff
@@ -10,8 +10,12 @@ export interface CheckboxProps
     OUIAProps {
   /** Additional classes added to the checkbox. */
   className?: string;
-  /** Additional classed added to the radio input */
+  /** Additional classed added to the checkbox input. */
   inputClassName?: string;
+  /** Flag to show if the checkbox label is wrapped on small screen. Will only apply if component prop is not specified. */
+  isLabelWrapped?: boolean;
+  /** Flag to show if the checkbox label is shown before the checkbox button. */
```

Comment:
nit - we refer to it as either "checkbox" or "checkbox input" in other prop descriptions

```suggestion
  /** Flag to show if the checkbox label is shown before the checkbox input. */
```